src/App.tsx

```diff
@@ -91,56 +92,58 @@ const App: FC = () => {
   }
 
   return (
-    <Routes>
-      <Route path="/" element={<Login />} />
-      <Route path="/login" element={<Login />} />
-      <Route
-        path="/setup"
-        element={
-          <Suspense fallback={<LoadingScreen />}>
-            <RequireSetup needsSetup={needsSetup}>
-              <LazySetup />
-            </RequireSetup>
-          </Suspense>
-        }
-      />
-      <Route
-        path="/home"
-        element={
-          <Suspense fallback={<SkeletonLoadingScreen />}>
-            <RequireAuth>
-              <Layout>
-                <LazyHome />
-              </Layout>
-            </RequireAuth>
-          </Suspense>
-        }
-      />
-      <Route
-        path="/apps"
-        element={
-          <Suspense fallback={<SkeletonLoadingScreen />}>
-            <RequireAuth>
-              <Layout>
-                <LazyApps />
-              </Layout>
-            </RequireAuth>
-          </Suspense>
-        }
-      />
-      <Route
-        path="/settings"
-        element={
-          <Suspense fallback={<SkeletonLoadingScreen />}>
-            <RequireAuth>
-              <Layout>
-                <LazySettings />
-              </Layout>
-            </RequireAuth>
-          </Suspense>
-        }
-      />
-    </Routes>
+    <NextUIProvider>
```

Comment:
Please add the Provider to the `index.tsx` instead (where the other Providers are located

Reply:
Thanks, it felt wrong :)